Docs/topics/discord/faq.md

```diff
@@ -11,32 +11,58 @@ Yes! On August 26th, 2023, we launched LittleBigRefresh. If you'd like to connec
 
-Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. Additionally, PSN will only work on PSPs running a CFW based on the latest firmware (6.61 is the latest official version as of writing) so make sure your custom firmware is up to date.
+Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. Additionally, PSN will only work on PSPs running a CFW based on the latest firmware, of which 6.61 is the latest as of writing, so make sure your custom firmware is up-to-date.
 
 If you still can't connect, try asking for help in <#1229318345891778581>!
 
-## My PSP can't connect to my WiFi network, is there any way to still connect?
+## My PSP can't connect to my Wi-Fi network, is there any way to still connect?
 
-The PSP does not support modern WPA/WPA2 networks.
+The PSP does not support modern WPA2 networks.
 
-You can setup a 2.4Ghz hotspot on your phone or computer with no password, and then use that on your PSP. You can also set up a WEP network if you want to use a password, although you should still use caution as it's still not as secure as WPA2.
+You can set up a 2.4Ghz hotspot on your phone or computer with no password, and then use that on your PSP. You can also set up a WEP network if you want to use a password, although you should still exercise caution as WPA is still not as secure as WPA2, you should only have that Wi-Fi network on when you are online.
 
 ## The PSP web browser crashes in Adrenaline when trying to go to a website!
 
-You should try disabling the `reScaler` plugin, as it conflicts with the PSP web browser.
+Try disabling the `reScaler` plugin, as it conflicts with the PSP web browser.
+
+## I get a 403 on connecting!
+
+Make sure you have an account registered [on our website](<https://lbp.littlebigrefresh.com/>).
+
+## Whenever I press the button to accept the EULA, nothing happens!
+
+Make sure RPCN is enabled in RPCS3's settings, the network is connected, and you have an RPCN account, you can find the RPCN setting in the "Network" tab.
+
+## I cant view/publish levels, and the game acts strangely on LBP1 and LBP3, even though I can connect fine!
+
+You likely forgot to use AutoDiscover, which is required on LBP1 and LBP3.
+
+## I can't log in on the site, and I am not receiving any password reset requests!
+
+Your account is likely unactivated, make sure you are able to log in from LBP before trying to log in on the site.
+
+## When I open the patcher, I get strange warnings about cool levels, and I don't see the GUI!
+
+You downloaded Refresh, not Refresher, please read one of our patching guides in <#1144875410786623568>.
+
+## On RPCS3, my game constantly diverges/I cant connect to anyone!
+
+In RPCS3 options, make sure to enable UPnP, you can find this setting in the "Network" tab in RPCS3's config.
```

Comment:
maybe mention xfloat?

Reply:
i meant to, but forgot after i wrote out the first bit about UPnP, will add that